### PR TITLE
Add Photoshop case study

### PR DESCRIPTION
**Pull request by tomayac**
Changes proposed in this pull request:

- Add Photoshop case study.
- This was reviewed in a [Google doc](https://docs.google.com/document/d/1Rko3TPHIwWvRy4dl6ST0gSRUpKVgeZWVsDvCIud4SeA/edit).
- ~Pending a huge sample `.psd` file. (Else, we can point people at a resource like [Freepik](https://www.freepik.com/free-psd/3d-camera-man-character-sit-big-camera_22821920.htm). The linked example is 234.7MB big.)~

### CODE REVIEW

**Comment by tomayac** [comment]
@rachelandrew After Oliver's approval and your happened review in the [Google doc](https://docs.google.com/document/d/1Rko3TPHIwWvRy4dl6ST0gSRUpKVgeZWVsDvCIud4SeA/edit#heading=h.fep53c1ahw2u), this is now good to go here. Thank you!

File: site/en/blog/how-photoshop-solved-working-with-files-larger-than-can-fit-into-memory/index.md
Comment (rachelandrew) [inline site/en/blog/how-photoshop-solved-working-with-files-larger-than-can-fit-into-memory/index.md:8]: I suggest adding a different thumbnail (using `thumbnail:`) here, if you look at the listing preview the image is not great. https://pr-5717-static-dot-dcc-staging.uc.r.appspot.com/blog/

File: site/en/blog/how-photoshop-solved-working-with-files-larger-than-can-fit-into-memory/index.md
Comment (rachelandrew) [inline site/en/blog/how-photoshop-solved-working-with-files-larger-than-can-fit-into-memory/index.md:6]:
```suggestion
date: 2023-03-23
```

File: site/en/blog/how-photoshop-solved-working-with-files-larger-than-can-fit-into-memory/index.md
Comment (rachelandrew) [inline site/en/blog/how-photoshop-solved-working-with-files-larger-than-can-fit-into-memory/index.md:37]:
```suggestion
As a historical note, the original problem that Photoshop needed to solve was editing print resolution files on early versions of macOS, as low as 1&nbsp;MB for the OS and all applications. A 300&nbsp;dpi full page image in CMYK is approximately 32&nbsp;MB uncompressed.
```

Comment (rachelandrew) [reply site/en/blog/how-photoshop-solved-working-with-files-larger-than-can-fit-into-memory/index.md:37]: As per styleguide https://developers.google.com/style/units-of-measure

**Review by rachelandrew — APPROVED** [review]
Approved but please check the notes/fixes below before merging.

File: site/en/blog/how-photoshop-solved-working-with-files-larger-than-can-fit-into-memory/index.md
Comment (tomayac) [inline site/en/blog/how-photoshop-solved-working-with-files-larger-than-can-fit-into-memory/index.md:10]:
```suggestion
thumbnail: image/8WbTDNrhLsU0El80frMBGE4eMCD3/tTlvAYtbWrg8xLqXdjua.png
authors:
```

File: site/en/blog/how-photoshop-solved-working-with-files-larger-than-can-fit-into-memory/index.md
Comment (rachelandrew) [reply site/en/blog/how-photoshop-solved-working-with-files-larger-than-can-fit-into-memory/index.md:8]: That looks better, please merge this when you are ready.

Comment (tomayac) [reply site/en/blog/how-photoshop-solved-working-with-files-larger-than-can-fit-into-memory/index.md:8]: Thanks for the love for the detail. I didn't notice this because I only ever saw the page itself, not the overview.

File: site/en/blog/how-photoshop-solved-working-with-files-larger-than-can-fit-into-memory/index.md
Comment (tomayac) [inline site/en/blog/how-photoshop-solved-working-with-files-larger-than-can-fit-into-memory/index.md:21]:
```suggestion

(This article is also available in form of a video.)

{% YouTube id="Y40vMQap9fs", startTime="695" %}
```

File: site/en/blog/how-photoshop-solved-working-with-files-larger-than-can-fit-into-memory/index.md
Comment (tomayac) [inline site/en/blog/how-photoshop-solved-working-with-files-larger-than-can-fit-into-memory/index.md:24]:
```suggestion
{% YouTube id="Y40vMQap9fs", startTime="695" %}

```

File: site/en/blog/how-photoshop-solved-working-with-files-larger-than-can-fit-into-memory/index.md
Comment (tomayac) [inline site/en/blog/how-photoshop-solved-working-with-files-larger-than-can-fit-into-memory/index.md:7]:
```suggestion
date: 2023-06-05
# updated: 2023-06-05
```

**Pull request merged**
